### ISSUE TITLE
Google sheets template is case sensitive

### ISSUE THREAD

**Issue by MokeEire** [nstrayer/datadrivencv#20]
The Google Sheets template indicates that users should put "current" for their current position.  This results in the following error message when running `build_network_logo(CV$entries_data)`:

```
Error in start:end : NA/NaN argument
In addition: Warning message:
In eval_tidy(xs[[j]], mask) : NAs introduced by coercion
```

However if you change "current" to "Current", there is no issue.  A change to the template, or `build_network_logo()` would fix this

**Comment by nstrayer** [comment]
Interesting! Can you try swapping your printing function code the new version in the branch  "improved_date_handling"? Specifically [these lines](https://github.com/nstrayer/datadrivencv/blob/improved_date_handling/inst/templates/CV_printing_functions.R#L59-L84). I tried to make this not case sensitive by converting all to lowercase but I probably messed it up.

**Comment by nstrayer** [comment]
I believe this is fixed in the newest version thanks to #25. Test it out and re-open if the issue persists! 

You can update to the newest version without wiping your rmd with

```
devtools::install_github("nstrayer/datadrivencv")
datadrivencv::use_datadriven_cv(
  full_name = ...
  which_files = c("cv_printing_functions.r")
)
```

**Comment by MokeEire** [comment]
Yep certainly seems fixed!
